### PR TITLE
if size is wrong string put it as small

### PR DESCRIPTION
**Pull request by a-c-sreedhar-reddy**
## Summary
#27224

## Changelog
If its a string and not small or medium make it as small.

### CODE REVIEW

File: Libraries/Components/ActivityIndicator/ActivityIndicator.js [hunk 1]
```diff
@@ -101,10 +106,7 @@ const ActivityIndicator = (props: Props, forwardedRef?: any) => {
   return (
     <View
       onLayout={onLayout}
-      style={StyleSheet.compose(
-        styles.container,
-        style,
-      )}>
```
[the remark this answers is not in the document]
Comment (a-c-sreedhar-reddy) [reply Libraries/Components/ActivityIndicator/ActivityIndicator.js]: changed

**Comment by TheSavior** [comment]
Christoph, we used to have validation on all props by using the prop-types package. We removed that for perf, complexity, and app size reasons. If we add validation back to all props like we are for this one we’d be back in that situation. Do you think this prop is special and worth validating more than other props so it would be reasonable to do the validation for this?

The other approach is that we recommend people use a type system like Flow or TypeScript in order to get that validation and safety. What are your thoughts?

**Comment by a-c-sreedhar-reddy** [comment]
How can I use Typescript in my RN project. I think docs are little out dated.

**Comment by cpojer** [comment]
Closing it based on Eli's comments. Let's rely on the type checker for this.

**Pull request closed without merging**